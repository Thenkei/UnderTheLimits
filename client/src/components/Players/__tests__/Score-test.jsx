// import React from 'react';
// import renderer from 'react-test-renderer';

// import Score from '..';

it('renders correctly', () => {
  // const tree = renderer.create(
  //   <Score
  //     players={[
  //       { id: 1, name: 'Player1', score: '120' },
  //       { id: 2, name: 'Player2', score: '200' },
  //     ]}
  //   />,
  // ).toJSON();
  // expect(tree).toMatchSnapshot();
});
