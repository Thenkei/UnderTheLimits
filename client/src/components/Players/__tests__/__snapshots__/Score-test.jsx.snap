// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<dl>
  <div>
    <dt>
      <div
        className="MuiChip-root-1 MuiChip-outlined-10"
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        role="button"
        style={
          Object {
            "marginTop": "5px",
          }
        }
        tabIndex={-1}
      >
        <div
          className="MuiAvatar-root-26 MuiChip-icon-17"
        >
          <img
            className="MuiAvatar-img-28"
            src="http://localhost/avatars/Player1.png"
          />
        </div>
        <span
          className="MuiChip-label-20"
        >
          Player1
        </span>
      </div>
    </dt>
  </div>
  <div>
    <dt>
      <div
        className="MuiChip-root-1 MuiChip-outlined-10"
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        role="button"
        style={
          Object {
            "marginTop": "5px",
          }
        }
        tabIndex={-1}
      >
        <div
          className="MuiAvatar-root-26 MuiChip-icon-17"
        >
          <img
            className="MuiAvatar-img-28"
            src="http://localhost/avatars/Player2.png"
          />
        </div>
        <span
          className="MuiChip-label-20"
        >
          Player2
        </span>
      </div>
    </dt>
  </div>
</dl>
`;
